docs/dev_script_usage.md

```diff
@@ -0,0 +1,75 @@
+# Using `scripts/dev` for Development Tasks
+
+Developers usually interact with `scripts/dev` as a frontend to
+various development commands instead of invoking them directly.
+
+## Available Commands
+
+```console
+$ scripts/dev
+Please provide a task to run:
+
+scripts/dev build             # Build the Go application
+scripts/dev db:clean          # Deletes all rows from all tables
+scripts/dev db:migrate        # Runs database migrations and wait for them to complete
+scripts/dev db:recreate       # Destroys the database container and recreates it
+scripts/dev db:seed           # Load development dataset
+scripts/dev docker:sweep      # Delete all dangling volumes
+scripts/dev down              # Stops all services in the project
+scripts/dev gql               # Generate code from GraphQL schema
+scripts/dev hosts:check       # Verify that hosts for local development are configured
+scripts/dev lint              # Run all linters and checks managed by pre-commit
+scripts/dev list              # List available tasks
+scripts/dev minio:clean       # Mark all files in minio as clean (no viruses found)
+scripts/dev minio:infected    # Mark all files in minio as infected (virus found)
+scripts/dev minio:pending     # Mark all files in minio as pending (waiting for scan)
+scripts/dev prereqs           # Check to see if the app's prerequisites are installed
+scripts/dev reset             # Resets application to an empty state
+scripts/dev restart           # Restart the specified container
+scripts/dev tailscale         # Run app and expose to other machines over Tailscale
+scripts/dev test              # Run all tests in parallel
+scripts/dev test:go           # Runs Go tests
+scripts/dev test:go:long      # Runs Go tests, including long ones
+scripts/dev test:go:only      # Run targeted Go tests (pass path to package folder as additional options)
+scripts/dev test:js           # Run JS tests (pass path to scope to that location)
+scripts/dev test:js:named     # Run JS tests with a matching name (pass needle as additional option)
+scripts/dev up                # Starts all services in the project
+scripts/dev up:backend        # Starts all services except the frontend (runs more quickly, if frontend isn't needed)
+scripts/dev up:backend:watch  # Starts all services in the foreground except the frontend (runs more quickly, if frontend isn't needed)
+scripts/dev up:watch          # Starts all services in the project in the foreground
+```
+
+
+## Starting the application
```

Comment:
It may be worth linking to this header from the `dev_environment_setup.md` file as a "Sweet, you're done setting things up. Lets try running it!". Thoughts?

Reply:
Added in https://github.com/CMSgov/easi-app/pull/1415/commits/1b7930f3e7f011ee516593b87a299717a2cbacc1.